packages/landingpage/pages/[lang]/library/patterns/index.tsx

```diff
@@ -45,39 +35,35 @@ const PatternsPage: NextPage<PatternsPageProps> = ({
   codeStrings,
 }) => {
   const { t } = useTranslation();
-  //TODO: Remove this  check when we implemented the first official pattern
-  if (!inDevEnvironment) {
-    return <Custom404 />;
-  }
 
   return (
     <Page title={[t('common.meta.library')]}>
-      <div className={styles.header}>
-        <h5 className="title-m">Lorem ipsum dolor sit amet.</h5>
-        <h1 className="header-d3">
-          <b>Design Pattern</b>
-        </h1>
-        <p className="body-l">
-          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
-          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
-          sed diam voluptua.
-        </p>
-        <div className={styles.divider} />
+      <div style={{ marginBottom: '8rem' }}>
+        <div className={styles.header}>
+          <h5 className="title-m">{t('designPatterns.header.top')}</h5>
+          <h1 className="header-d3">
+            <b>{t('designPatterns.header.title')}</b>
+          </h1>
+          <p className="body-l">{t('designPatterns.header.subtitle')}</p>
+          <div className={styles.divider} />
+        </div>
+        <PreviewBox
+          title={t('designPatterns.loginPattern.title')}
+          id="login"
+          description={t('designPatterns.loginPattern.description')}
+          previewComponent={<Login />}
+          highlightedCode={highlightedCodes.login}
+          rawCode={codeStrings.login}
+          requiresJS={true}
```

Comment:
Why does this pattern require JS? 🤔

Reply:
I found inspiration in Tailwind CSS and appreciated its unique approach. However, considering it's not a part of the original design and given the nature of our inovex Elements, mentioning this might be redundant or obvious. Do you think it should be omitted?

![image](https://github.com/inovex/elements/assets/81302108/49b84383-78cd-4c07-ac8d-40ad94cbf7da)